clients/stellar-relay-lib/resources/config/testnet/stellar_relay_config_sdftest2.json

```diff
@@ -1,6 +1,6 @@
 {
   "connection_info": {
-    "address": "34.235.168.98",
+    "address": "3.82.209.201",
```

Comment:
Even if there are issues with sdftest1 at the moment, maybe it would make more sense not to _replace_ sdftest1 with sdftest2 but to create a new file next to it?

Reply:
done. this sdftest2 is added

Reply:
done